.github/workflows/gosec.yml

```
@@ -0,0 +1,38 @@
+name: Run Gosec
+on:
+  pull_request:
+    paths:
+      - '**/*.go'
+      - 'go.mod'
+      - 'go.sum'
+  push:
+    branches:
+      - main
```

@odeke-em would it be valuable to run this on feature and release branches? e.g.:

```suggestion
      - main
      - 'feature/*'
      - v0.37.x
      - v0.34.x
```

---

Sounds good to me! Thank you @thanethomson

---

Thank you, I committed this suggestion @thanethomson.